### PR TITLE
translate docs/zh/operations/table_engines/merge.md

### PR DESCRIPTION
**Pull request by neverlee**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

For changelog. Remove if this is non-significant change.

Category (leave one):
- Other

Short description (up to few sentences):

translate docs/zh/operations/table_engines/merge.md

Detailed description (optional):
I need some help for this:
If the `WHERE/PREWHERE` clause contains conditions for the `_table` column that do not depend on other table columns (**as one of the conjunction elements, or as an entire expression**),

### CODE REVIEW

**Comment by blinkov** [comment]
cc @zhang2014 @sundy-li

File: docs/zh/operations/table_engines/materializedview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# MaterializedView
+# 物化视图
 
-Used for implementing materialized views (for more information, see [CREATE TABLE](../../query_language/create.md)). For storing data, it uses a different engine that was specified when creating the view. When reading from a table, it just uses this engine.
+Used for implementing materialized views (详情参见 [CREATE TABLE](../../query_language/create.md)). For storing data, it uses a different engine that was specified when creating the view. When reading from a table, it just uses this engine.
```
Comment (zhang2014) [inline docs/zh/operations/table_engines/materializedview.md]: 需要翻译

Comment (neverlee) [reply docs/zh/operations/table_engines/materializedview.md]: 尴尬，add错分支了。

**Comment by zhang2014** [comment]
LGTM

**Pull request merged**